Formula/miranda.rb

```diff
@@ -0,0 +1,63 @@
+class Miranda < Formula
+  desc "Lazy, purely functional programming language"
+  homepage "http://miranda.org.uk/"
+  url "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-2066-src.tgz"
+  version "2.066"
+  sha256 "521c281e8c2fde87a2cd7c41d9677daa09514debb71435efc08ff3a7c20016eb"
+
+  depends_on "byacc" => :build
+
+  def install
+    # Make tasks are required to run sequentially
```

Comment:
Why? Can an issue be submitted upstream for this?

Reply:
As I explained earlier upstream in the conventional sense does not exist and this is built upon source code dump that hasn't been actively maintained for a long time.

Reply:
As I explained earlier upstream in the conventional sense does not exist and this is built upon source code dump that hasn't been actively maintained for a long time.

Reply:
If there is no upstream this software is not maintained and not a good fit for Homebrew/core, sorry (https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff)

Reply:
Just to make it super clear:

>this is built upon source code dump that hasn't been actively maintained for a long time.

It is my assumption. I can't and do not know for sure due to it being closed source until recently.

Maybe the author still occasionally works on it and accepts patches but I'm not willing to start contributing by mailing patches around without any transparency.

Again, I just wanted to make it easier for people to play with it but if this is a bad place to host it I'll move it to personal tap. I'm keeping it open and waiting for your final decision.

Reply:
Ok, will pass on this for now given the above, thanks anyway!